### PR TITLE
Gl 203

### PR DESCRIPTION
**Pull request by montu1996**
FIX GL-203

### CODE REVIEW

File: app/helper/RTMediaLikeNotification.php [hunk 1]
```diff
@@ -81,7 +81,10 @@ function format_like_notifications( $params ) {
 
 			$rtmedia_id = rtmedia_id( $post_id );
 			$liked_list = $this->fetch_media_like_stats( $rtmedia_id );
-			$liked_by   = bp_core_get_username( $liked_list[0]->user_id );
+                        $liked_by   = '';
```
Comment (thrijith) [inline app/helper/RTMediaLikeNotification.php]: check where `$liked_by` is used make sure given default value doesn't cause other implications, also align variable.

File: app/main/RTMedia.php [hunk 2]
```diff
@@ -1395,13 +1395,15 @@ function get_rtmedia_permalink( $id ) {
...
 	} else {
-		if ( function_exists( 'bp_get_groups_root_slug' ) && 'group' === $media[0]->context ) {
+		if ( is_object( $media[0] ) && ! empty( $media[0] ) && isset( $media[0]->context )  && function_exists( 'bp_get_groups_root_slug' ) && 'group' === $media[0]->context ) {
 			$parent_link = get_rtmedia_group_link( $media[0]->context_id );
 		} else {
 			// check for global album
 			$parent_link = parentlink_global_album( $id );
 			if ( '' === $parent_link ) {
-				$parent_link = get_rtmedia_user_link( $media[0]->media_author );
+                            if( ! empty( $media ) && isset( $media[0] ) && ! empty( $media[0] ) && isset( $media[0]->media_author ) ) {
```
Comment (milindmore22) [inline app/main/RTMedia.php]: space before opening parenthesis. avoid excessive conditions .

File: app/main/RTMedia.php [hunk 3]
```diff
@@ -1395,13 +1395,15 @@ function get_rtmedia_permalink( $id ) {
 			}
 		}
 	} else {
-		if ( function_exists( 'bp_get_groups_root_slug' ) && 'group' === $media[0]->context ) {
+		if ( is_object( $media[0] ) && ! empty( $media[0] ) && isset( $media[0]->context )  && function_exists( 'bp_get_groups_root_slug' ) && 'group' === $media[0]->context ) {
```
Comment (milindmore22) [inline app/main/RTMedia.php]: avoid excessive conditions check

File: templates/media/media-gallery-item.php [hunk 4]
```diff
@@ -33,7 +33,10 @@ class="<?php echo esc_attr( apply_filters( 'rtmedia_gallery_list_item_a_class',
 			global $rtmedia_query;
 
 			$alt_text      = rtmedia_image_alt( false, false );
-			$rtmedia_media = $rtmedia_query->rtmedia;
+			$rtmedia_media = '';
+                        if ( ! empty( $rtmedia_query ) && isset( $rtmedia_query->rtmedia ) ) {
```
Comment (milindmore22) [inline templates/media/media-gallery-item.php]: Used ternary operator to wrap this in one line.

Comment (montu1996) [reply templates/media/media-gallery-item.php]: This changes approved by @chandrapatel . LINK: https://github.com/rtMediaWP/rtMedia/pull/1055 @milindmore22

Comment (milindmore22) [reply templates/media/media-gallery-item.php]: why are those changes in this PR?

**Pull request closed without merging**